integration-tests/.gitignore

```diff
@@ -1,2 +1,2 @@
-# Symlink to virtualenv
-.venv
```

Comment:
Yeah, it's in the wrong place. The real one is one level up in the base `.gitignore`. I just hadn't noticed it was still here until I went to add the exclusions for the directory with the XML reports.